src/core/lib/gprpp/ref_counted_ptr.h

```diff
@@ -103,6 +103,12 @@ class RefCountedPtr {
     if (value_ != nullptr) value_->Unref();
   }
 
+  void swap(RefCountedPtr& other) {
+    T* tmp = value_;
```

Comment:
nit and optional: `std::swap(value_, other.value_)` is header-only and shorter IMO.

Reply:
Done.